test/cqlpy/test_map_key_selection.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020-present ScyllaDB
```

Comment:
Off-by-4

Reply:
Well, this feature was due to be implemented in 2020 (when Cassandra added it), so we can pretend we did it in 2020 too :-)